os/env.go

```diff
@@ -0,0 +1,32 @@
+package os
+
+import "os"
+
+// EnvironmentRepository ...
+type EnvironmentRepository interface {
+	SetVariable(key string, value string) error
```

Comment:
```suggestion
	SetVariable(key, value string) error
```

Reply:
I'm not a fan of this suggested syntax.

Reply:
+1, I still get confused when I read something like this in function parameter lists.

Reply:
https://github.com/mvdan/gofumpt#extra-rules-behind--extra
There is an "extended" formatter which has this as en extra option. 

That being said I still think it makes the code less readable so I'd be happy to write type for all params

Reply:
Let's agree to use the full syntax than, I'm okay with it.